raycicmd/rayci_pipeline.go

```diff
@@ -45,6 +45,7 @@ var (
 		"instance_type", "queue", "job_env", "tags",
 		"docker_publish_tcp_ports", "docker_network",
 		"mount_buildkite_agent", "mount_windows_artifacts",
+		"allow_dependency_failure",
```

Comment:
the test is good, but could you help add the comment from https://github.com/ray-project/rayci/pull/209/files ?

specifically, group the keys that are going to be inherited together, and the keys that are going to be dropped in another group.

Reply:
done!